Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/InformationItemList.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from "react";
+import InformationItem from "./InformationItem";
+import Button from "@material-ui/core/Button";
+
+class InformationItemList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      iItems: [],
+      iItemCur: {
+        body: "",
+        id: 0,
+      },
+    };
+    this.addIItem = this.addIItem.bind(this);
+  }
+
+  addIItem(e) {
+    e.preventDefault();
+    const newIItem = this.state.iItemCur;
+    const newIItems = [...this.state.iItems, newIItem];
+
+    this.setState({
+      iItems: newIItems,
+      iItemCur: {
+        body: "",
+        id: this.state.iItemCur.id + 1,
+      },
+    });
+  }
+
+  handleDelete = (iItemID) => {
+    const iItems = this.state.iItems.filter((q) => q.id !== iItemID);
+    this.setState({ iItems });
+  };
+
+  updateIItem = (iItemID, iItemBody) => {
+    const newIItems = [...this.state.iItems];
+    
+    //does this even save stuff to the backend?
+    newIItems.forEach(element => {if(element.props.id === iItemID){element.props.body = iItemBody}})
```

Comment:
Not yet, but this is where we will put the logic to save data to backend once we set it up.